src/importlinter/domain/fields.py

```diff
@@ -7,6 +7,21 @@
 FieldValue = TypeVar("FieldValue")
 
 
+def _validate_wildcard(expression: str) -> None:
```

Comment:
I wonder if we should be careful about using the same wildcard syntax used by ignored imports. For example, what would it actually mean if we listed `mypackage.**` in an independence contract? What about `mypackage.*.foo`?

I had been anticipating a much more limited set of expressions: i.e. just of the form `mypackage.*`.

Then again, if the functionality is already there, then perhaps it's counterproductive to special-case it. People may come up with use cases we hadn't thought of. What do you think?